api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/IntensiveMonitoringCohortQueries.java

```diff
@@ -2728,7 +2729,9 @@ public CohortDefinition getMI13DEN1() {
     CohortDefinition abandonedExclusionFirstLine =
         qualityImprovement2020CohortQueries.getPatientsWhoAbandonedTarvOnOnFirstLineDate();
 
-    CohortDefinition PrimeiraLinha = qualityImprovement2020CohortQueries.getUtentesPrimeiraLinha();
+    CohortDefinition PrimeiraLinha =
+        qualityImprovement2020CohortQueries.getUtentesPrimeiraLinha(
+            QualityImprovement2020CohortQueries.UtPrimeiraLPreposition.MI, "DEN");
```

Comment:
Don't abbreviate variable names
Use enums for flags instead of strings

Reply:
Resolved